vmdb/app/models/provider_foreman.rb

```diff
@@ -1,15 +1,20 @@
 require 'manageiq_foreman'
+
 class ProviderForeman < Provider
   has_one :configuration_manager,
           :foreign_key => "provider_id",
           :class_name  => "ConfigurationManagerForeman",
-          :dependent   => :destroy
+          :dependent   => :destroy,
+          :autosave    => true
```

Comment:
Turns out I can't when I move it into EMS.  Didn't want to do this, but kind of had to in order to propogate name and zone changes from the provider.  Turns out you *can't* delegate on real columns...since EMS has a name column, I couldn't delegate to provider.  I had to propogate for now.